proxy/http2/Http2CommonSession.cc

```diff
@@ -354,7 +355,8 @@ Http2CommonSession::do_process_frame_read(int event, VIO *vio, bool inside_frame
     }
 
     Http2ErrorCode err = Http2ErrorCode::HTTP2_ERROR_NO_ERROR;
-    if (this->connection_state.get_stream_error_rate() > std::min(1.0, Http2::stream_error_rate_threshold * 2.0)) {
+    if (this->connection_state.get_stream_error_rate() > std::min(1.0, Http2::stream_error_rate_threshold * 2.0) &&
+        !this->is_outbound()) {
```

Comment:
I wonder why we don't enforce the error rate check.

Reply:
As I remember, I was seeing lots of spurious errors in the outbound case.  Things that we were counting as errors, but weren't really.  So we were closing things unnecessarily.

Reply:
Why don't you fix the code that reports the spurious errors? I don't think we can ignore errors like this unless you can say real errors never happen in the outbound case.

Reply:
I added the stream error rate check on outbound streams.